js/components/SyriacLetter.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+
+/* SyriacLetter - this component renders the specified Syriac letter, keyed by its ID,
+ * based on the data in the "letters" table. note that the script, leading and trailing
+ * letter data is only stored here at present, and not available via the backend
+ * REST API.
+ */
+
+export const letters = [
```

Comment:
This also would be great stored as a separate json files that is imported into the component, or into the other one that references this data.

Reply:
Yeah, I've moved the list into letters.json, currently in the same folder (we can move it if that seems called for, but at the moment, it would be all by itself in a "data/" folder). I've updated the components and test cases to load the letters list from there.